### ISSUE TITLE
missing emm_private.h in sgx_rsrv_mem.cpp

### ISSUE THREAD

**Issue by 2-Ritual** [intel/linux-sgx#1043]
I work on an unsupported operating system OpenEuler22.03 and use it as centOS8. I think the required tools to build the Intel(R) SGX SDK are installed and they are the latest version. When I execute the command 'make sdk', the problem is the missing file 'emm_private.h'. Is the problem that the corresponding tool for my OS is not suitable?

**Comment by jbdelcuv** [comment]
Have you checked out all the submodules?
I believe that file is provided by the sgx-emm/emm_src submodule.

**Comment by olafhering** [comment]
Yes, it is in that submodule. How are the build rules supposed to use `external/sgx-emm/emm_src/include/emm_private.h`?
This command in your favorite  off-the-shelf operating system fails to find the header file:
````
git clone --recurse-submodules https://github.com/intel/linux-sgx.git x.git
pushd "$_"
bash download_prebuilt.sh
make sdk
````
